bait-files/src/main/java/com/google/sps/servlets/Request.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+import com.googlecode.objectify.Key;
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+
+@Entity
+public class Request {
+  @Id Long requestId;
+  String userId;
+  String status;
```

Comment:
What are the options here? Maybe this should be an enum.

Reply:
Do you mean for the @Id requestId? It can be Long, long, or a String but it's only given an auto-generated value if you use a Long.

Reply:
No,